tests/review-pr.test.sh

```
@@ -3030,6 +3361,332 @@ assert_grep "$REVIEW_PR" "printf 'REVIEW_CARRY RUN_ID=%s" \
+review_capture_ci_classification_head review_ci_authority_digest review_ci_json_member
+review_validate_trust_anchor"
+R47_MISSING="$(env -i PATH="$PATH" HOME="$HOME" \
+  UBERDEV_REVIEW_PLUGIN_ROOT="$REPO_ROOT/plugins/uberdev" \
+  R47_HELPERS="$R47_HELPERS" \
+  bash -c '
+    . "$UBERDEV_REVIEW_PLUGIN_ROOT/lib/review-fleet-args.sh" || exit 9
+    review_fleet_load_fence_library || exit 9
+    for fn in $R47_HELPERS; do
+      typeset -f "$fn" >/dev/null 2>&1 || printf "%s " "$fn"
+    done
+  ' 2>/dev/null)" || R47_MISSING="LOAD_FAILED"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)